minion/config_topic_config.go

```diff
@@ -18,6 +20,16 @@ type TopicConfig struct {
 	// IgnoredTopics are regex strings of topic names that shall be ignored/skipped when exporting metrics. Ignored topics
 	// take precedence over allowed topics.
 	IgnoredTopics []string `koanf:"ignoredTopics"`
+
+	// InfoMetric configures how the kafka_topic_info metric is populated
+	InfoMetric InfoMetricConfig `koanf:"exporter"`
+}
+
+type InfoMetricConfig struct {
+	// ConfigKeys configures optional topic configuration keys that should be exported
+	// as prometheus metric labels.
+	// By default "topic_name", "partition_count", "replication_factor" and "cleanup.policy" are exported
```

Comment:
```suggestion
	// By default only "cleanup.policy" is exported
```

I think you wanted to list all labels here, but it's kinda confusing since this config only refers to the configurations and a user might think they would need to add "topic_name" etc as configKey as well